docs/Inference-config.yml

```diff
@@ -0,0 +1,36 @@
+# Configuration detailing the staining prediction on label-free images
+
+# define the dataset you want to use the trained model for virtual staining,
+# the output formats required and predicted image quality metrics
+
+# point to directory where your trained model is saved
+model_dir: '/home/Translation_temp_2/'
+
+# directory with images on which you want to perform the prediction, the inference dataset
+image_dir: '/home/InferenceData/'
+
+# directory with all preprocessed data
+preprocess_dir: '/home/Processed_temp_1'
+
+# define inference dataset channels
+dataset:
+  input_channels: [2]   # label-free channel used for prediction by model
+  target_channels: [1]  # target image channel (fluorescence image) to compare how well the prediction worked
+#  slice_ids: [0]
```

Comment:
slice_ids should not be removed entirely _or_ not be commented out and have documentation